recipes/libmatio/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+./autogen.sh
+./configure --quiet --enable-shared --enable-mat73 --enable-extended-sparse --prefix="${PREFIX}" --with-zlib="${PREFIX}" --with-hdf5="${PREFIX}"
+make
+make install
```

Comment:
Ah, yes, familiar with this sort of stuff. Take a look at [this recipe]( https://github.com/conda-forge/vigra-feedstock/blob/master/recipe/build.sh#L79 ) to get an idea on how to address this sort of problem.